build.cake

```diff
@@ -180,6 +180,24 @@ Task("TestAwsAuthentication")
             );
         });
 
+Task("TestPlainAuthentication")
+    .IsDependentOn("Build")
+    .DoesForEach(
```

Comment:
There's only one matching `.csproj` file. Is there a simpler way to do this that doesn't involve looping over a single value?